storage/badger/operation/prefix.go

```diff
@@ -139,3 +140,19 @@ func b(v interface{}) []byte {
 		panic(fmt.Sprintf("unsupported type to convert (%T)", v))
 	}
 }
+
+// d is a reverse of b.
+// It decodes input bytes according to the return type T.
+// All types possible for T should be supported, hence no error should be possible provided the code compiles.
+func d[T uint64 | byte](bytes []byte) (ret T) {
```

Comment:
what is this used for (i don't see any actual usage)?  also, can you give the function a better name